README.md

```diff
@@ -101,6 +101,16 @@ lists = []
 $$);
 ```
 
+> [!NOTE]
+> The `lists` option should be configured based on the number of vectors. Below is a table to assist with your selection.
+
+| vectors Range | List Calculation Formula       | Example Result   |
+| ------------- | ------------------------------ | ---------------- |
+| <128k         | list = 1                       | 1                |
+| ≥128k and <2M | list = (2 * vectors) / 1000    | [256, 4000]      |
```

Comment:
`256 - 4000` looks better. User may think `[256, 4000]` as a python list instead of a closed interval

Reply:
I will explain below